Source/WebCore/platform/audio/gstreamer/AudioFileReaderGStreamer.cpp

```diff
@@ -319,19 +319,11 @@ void AudioFileReader::handleNewDeinterleavePad(GstPad* pad)
     }
     m_channels++;
 
-    static GstAppSinkCallbacks callbacks = {
-        nullptr, // eos
-        nullptr, // new_preroll
-        // new_sample
-        [](GstAppSink* sink, gpointer userData) -> GstFlowReturn {
-            return static_cast<AudioFileReader*>(userData)->handleSample(sink);
-        },
-#if GST_CHECK_VERSION(1, 20, 0)
-        // new_event
-        nullptr,
-#endif
-        { nullptr }
+    static GstAppSinkCallbacks callbacks;
+    callbacks.new_sample = [](GstAppSink* sink, gpointer userData) -> GstFlowReturn {
+        return static_cast<AudioFileReader*>(userData)->handleSample(sink);
     };
+
```

Comment:
This isn't ideal because the `new_sample` value gets overridden with the lambda address whenever this is called.

Maybe designated initializers could be used:
https://en.cppreference.com/w/cpp/language/aggregate_initialization#Designated_initializers